simulation_run.py

```diff
@@ -14,32 +14,23 @@
 
 import scenario
 import contributivity
-import data_splitting
 import fl_training
 import contributivity_measures
-import constants
 
 from timeit import default_timer as timer
 import numpy as np
+import matplotlib.pyplot as plt
 
+plt.close('all')
 
 #%% Create scenarii
 
-# Create a default scenario
-my_default_scenario = scenario.Scenario()
-
 # Create a custom scenario and comment the main scenario parameters (see scenario.py for more comments)
-my_custom_scenario = scenario.Scenario()
+my_custom_scenario = scenario.Scenario(is_quick_demo=True)
 my_custom_scenario.nodes_count = 3 # Number of nodes in the collaborative ML project simulated
 my_custom_scenario.amounts_per_node = [0.20, 0.30, 0.5] # Percentages of the data samples for each node
-my_custom_scenario.samples_split_option = 'Stratified' # If data are split randomly between nodes or stratified to be distinct (toggle between 'Random' and 'Stratified')
+my_custom_scenario.samples_split_option = 'Random' # If data are split randomly between nodes or stratified to be distinct (toggle between 'Random' and 'Stratified')
 my_custom_scenario.testset_option = 'Centralised' # If test data are distributed between nodes or stays a central testset (toggle between 'Centralised' and 'Distributed')
-my_custom_scenario.nb_epochs = constants.NB_EPOCHS
```

Comment:
I suggest to leave the line `my_custom_scenario.epoch_count = 15` here so that the user knows easily what can be configured in a custom scenario @RomainGoussault

Reply:
If we leave it here, the boolean `is_quick_demo` in the Scenario constructor will not work anymore. 

If we want to leave `my_custom_scenario.nb_epochs`, we wil have to re-work the `is_quick_demo` part